src/interfaces/utils-interfaces/form-type.ts

```diff
@@ -1,6 +1,6 @@
 import Vue from 'vue'
 
 export interface FormType extends Vue {
-  reset(): void;
-  validate(): boolean;
+  reset(): void
+  validate(): boolean
```

Comment:
no action required here, but i feel we have some interfaces with semicolons and some without. Should we go without, for all?

Reply:
Totally, i'm impartial. So we can agree going forward with this format!